app/src/main/java/com/anotherdeveloper/androidcourse/adapters/ContactsRecyclerViewAdapter.java

```diff
@@ -0,0 +1,163 @@
+package com.anotherdeveloper.androidcourse.adapters;
+
+import android.content.Context;
+import android.content.Intent;
+import android.graphics.Bitmap;
+import android.graphics.BitmapFactory;
+import android.graphics.drawable.Drawable;
+import android.support.annotation.NonNull;
+import android.support.v7.widget.RecyclerView;
+import android.util.Log;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+import com.anotherdeveloper.androidcourse.R;
+import com.anotherdeveloper.androidcourse.activities.A2;
+import com.anotherdeveloper.androidcourse.models.Contact;
+import com.squareup.picasso.Callback;
+import com.squareup.picasso.NetworkPolicy;
+import com.squareup.picasso.Picasso;
+import com.squareup.picasso.Target;
+
+import java.io.ByteArrayOutputStream;
+import java.util.ArrayList;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+/**
+ * Created by Marcin on 2017-12-20.
+ * :)
+ */
+
+public class ContactsRecyclerViewAdapter extends RecyclerView.Adapter<ContactsRecyclerViewAdapter.ViewHolder> {
+    private ArrayList<Contact> items;
+    private int itemLayout;
+    private Context context;
```

Comment:
Adapter właściwie nie powinien mieć przekazywanego kontekstu, bo zawsze możesz sobie go wyciągnąć z itemView, jak to robisz w linii 49